src/main/java/vazkii/botania/client/integration/nei/recipe/RecipeHandlerElvenTrade.java

```diff
@@ -152,4 +153,28 @@ public void loadUsageRecipes(ItemStack ingredient) {
 		}
 	}
 
+	// hide dummy recipes
+	private List<RecipeElvenTrade> filteredElvenTradeRecipes() {
+		List<RecipeElvenTrade> list = new ArrayList<>();
```

Comment:
It'd be nice to not copy the list on every invocation. Maybe use java Stream API or use guava's Iterables.filter. Either is OK imo. The first is the standard going forward, while the second offers the friendly Iterable interface so you can use it in a for each structrue as usual.